scenes/gameplay/map/checkpoint.gd

```diff
@@ -0,0 +1,24 @@
+extends Node2D
+
+export var sceneLoad = 1
```

Comment:
In GDScript it's a good practice tu use snake_case style for variables, functions, etc. 
So this should be `scene_load`.